Baby Monitor/Source Files/Services/MediaPlayer/MicrophoneAmplitudeTracker.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  MicrophoneAmplitudeTracker.swift
+//  Baby Monitor
+
+import Foundation
+import AudioKit
+
+/// A tracker that detects amplitude from the microphone.
+protocol MicrophoneAmplitudeTracker: Any {
+
+    /// The current amplitude with values from 0 to 1.
+    var amplitude: Double { get }
+
+    /// The loudness factor customly counted in order to get poercentage-like result.
```

Comment:
There is a typo *p~~o~~ercentage*. Btw. I think the comment can be simplified to increase the signal to noise ratio.

```suggestion
    /// The computed loudness factor as a percentage, from 0 to 100 inclusive.
```